ql/src/test/queries/clientpositive/file_with_delimiter.q

```diff
@@ -0,0 +1,22 @@
+create external table test(code string,name string)
+ROW FORMAT SERDE
+   'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe'
+ WITH SERDEPROPERTIES (
+   'field.delim'='\t')
+ STORED AS INPUTFORMAT
+   'org.apache.hadoop.mapred.TextInputFormat'
+ OUTPUTFORMAT
+   'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+   location '${system:test.tmp.dir}/test'
+ TBLPROPERTIES (
+   'skip.header.line.count'='1',
+   'textinputformat.record.delimiter'='|');
+
+
+   LOAD DATA LOCAL INPATH '../../data/files/header_footer_table_4/0003.txt' INTO TABLE test;
```

Comment:
indentation: why are some lines indented?

Reply:
nit: mixed-case is weird: "Select", "create", "STORED", please pick one and stick to it, no matter if it's full uppercase or lowercase

Reply:
Okay, will remove extra space and convert to full uppercase. Thanks